docs/pages.jl

```diff
@@ -16,8 +16,10 @@ pages = Any["Home" => "index.md",
                                            "catalyst_applications/homotopy_continuation.md",
                                            "catalyst_applications/nonlinear_solve.md",
                                            "catalyst_applications/bifurcation_diagrams.md"],
-            "Inverse Problems" => Any["inverse_problems/parameter_estimation.md",
-                                      "inverse_problems/petab_ode_param_fitting.md",
-                                      "inverse_problems/structural_identifiability.md"],
-            "FAQs" => "faqs.md",
```

Comment:
You removed the FAQs?

Reply:
Sorry, seemed to have gotten messed up in the latest merge. Fixed now.